lib/pizza.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/prices'
+
+class Pizza
+  include Prices
+  attr_reader :size, :type, :crust
+
+  def initialize(size, type, crust)
```

Comment:
Do these need to be stored like this? Can we have a `@full_order` ivar instead that contains a hash?